external/js/cothority/src/byzcoin/contracts/spawner-instance.ts

```diff
@@ -58,84 +86,18 @@ export default class SpawnerInstance {
     /**
      * Initializes using an existing coinInstance from ByzCoin
      *
-     * @param bc
-     * @param iid
+     * @param bc an initialized byzcoin RPC instance
+     * @param iid the instance-ID of the spawn-instance
      */
     static async fromByzcoin(bc: ByzCoinRPC, iid: InstanceID): Promise<SpawnerInstance> {
```

Comment:
add the wait/repeat parameter with default values

Reply:
Added it to all the `Instance`s.